de-lib/src/main/java/org/iplantc/de/theme/base/client/desktop/DesktopMessages.properties

```diff
@@ -29,3 +29,4 @@ saveSettings = Setting saved.
 savingSession = Saving Session
 savingSessionWaitNotice = Saving current session, please wait...
 requestHistoryError = Unable to retrieve request history!
+waitTime = Display warning about wait times for submitting HPC apps.
```

Comment:
Thats the text used in the User Preferences dialog to enable / disable message about long wait times.